packages/owl-bot/test/fixtures/push.json

```diff
@@ -0,0 +1,246 @@
+{
+  "action": "push",
+  "ref": "refs/heads/master",
+  "before": "9332efbb42e9d7d9c517619a54ddc12c7cadc4e9",
+  "after": "ab335197f41077988cae4b2d386cd388275b5d19",
+  "repository": {
+    "id": 203461499,
+    "node_id": "MDEwOlJlcG9zaXRvcnkyMDM0NjE0OTk=",
+    "name": "googleapis-gen",
+    "full_name": "googleapis/googleapis-gen",
+    "private": false,
+    "owner": {
+      "name": "googleapis",
+      "email": null,
+      "login": "googleapis",
+      "id": 16785467,
+      "node_id": "MDEyOk9yZ2FuaXphdGlvbjE2Nzg1NDY3",
+      "avatar_url": "https://avatars.githubusercontent.com/u/16785467?v=4",
+      "gravatar_id": "",
+      "url": "https://api.github.com/users/googleapis",
+      "html_url": "https://github.com/googleapis",
+      "followers_url": "https://api.github.com/users/googleapis/followers",
+      "following_url": "https://api.github.com/users/googleapis/following{/other_user}",
+      "gists_url": "https://api.github.com/users/googleapis/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/googleapis/starred{/owner}{/repo}",
```

Comment:
This fixture immediately becomes useful once we implement the real-time endpoints, I figured let's just leave it.